easybuild/tools/filetools.py

```diff
@@ -823,6 +831,8 @@ def download_file(filename, url, path, forced=False, trace=True):
     used_urllib = std_urllib
     switch_to_requests = False
 
+    wait_time_secs = 1
```

Comment:
```suggestion
    wait_time_secs = 8
```
Since the rate limiting is so aggressive, i think we can skip the first few doublings. (or make a configurable parameter, but i don't think that's necessary since the limit is due to github, and shouldn't change much from site to site.